x/poolmanager/router.go

```diff
@@ -559,9 +560,11 @@ func (k Keeper) ListPoolsByDenom(
 
 		var poolsByDenom []types.PoolI
 		for _, pool := range currentModulePools {
+			// If the pool is incorrectly implemented and we can't get the PoolDenoms
+			// skip the pool.
 			poolDenoms, err := poolModule.GetPoolDenoms(ctx, pool.GetId())
 			if err != nil {
-				return nil, err
+				continue
```

Comment:
Should we add a debug log here so we can tell what is being skipped in the future without having to use debugger?

Reply:
Do we do logs in queries? I don't actually know